SplashScreen: Added Activity

rakshitambi7a opened the pull request:
Created the splash screen

shubhh7935 commented:
Please follow the Pull Request Guidelines for better understanding of features. 
"
**Pull Request Guidelines**
Create a pull request from bash contributor/<branch-name> → main. Follow the pull request template:

## Description
<!-- Describe the changes in this pull request -->

## ✅ Checklist
- [ ] Code follows project guidelines
- [ ] Added necessary documentation
- [ ] Added tests (if applicable)
- [ ] No merge conflicts

## Screenshots (if applicable)
<!-- Add screenshots here -->
"

rakshitambi7a commented:
## Description
Created a Splash Screen Activity as Launcher Activity

## ✅ Checklist
- [x] Code follows project guidelines
- [x] Added necessary documentation
- [x] Added tests (if applicable)
- [x] No merge conflicts

## Screenshots (if applicable)
![image](https://github.com/user-attachments/assets/5d30f91d-b009-418d-ac56-3f311cef667d)
